### DISCUSSION TITLE
Integrate with Pitch

### DISCUSSION THREAD

**Discussion by fmerian** [documenso/documenso#241] [Ideas]
I use [Pitch](https://pitch.com) to create pitch decks. Before sending them to potential investors, I ask recipients to sign an NDA by email.

Instead, I'd like to embed Documenso so potential investors can:

1. open the Pitch presentation,
2. sign the NDA,
3. view the presentation.

**Comment by Mythie** [comment]
It sounds like an easy way to solve this would be post-signing actions.

e.g. I a document author define an action to view the gated content (pitch deck) and then you the signee can see an additional button after signing that would say "View Pitch Deck" or similar.

**Comment by ElTimuro** [comment]
- While the concept is nice, it's a go-go to ask an investor for an NDA before sending a deck
- Other cases could work Product Details for sales, medical compliance waivers etc.
